src/Symfony/Component/Notifier/Bridge/Lox24/tests/Lox24TransportFactoryTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Notifier\Bridge\Lox24\Lox24TransportFactory;
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\Dsn;
+
+/**
+ * @author Andrei Lebedev <andrew.lebedev@gmail.com>
+ */
+class Lox24TransportFactoryTest extends TestCase
```

Comment:
Please extend the Abstract Test case, like the other bridges

Reply:
done